win32/src/threadapi_win32.c

```diff
@@ -83,3 +83,8 @@ void ThreadAPI_Sleep(unsigned int milliseconds)
 {
     Sleep(milliseconds);
 }
+
+uint32_t ThreadAPI_GetCurrentId(void)
+{
+    return (uint32_t)GetCurrentThreadId();
```

Comment:
>(uint32_t) [](http://example.com/codeflow?start=11&length=10)

me no likey this cast. I actually looked at the header first and was thinking "well, the type of the thread Id can be vaaastly different from OS to OS"

Reply:
GetCurrentThreadId returns a DWORD which is unsigned long which sounds reasonable to cast to unsigned int.